### PR TITLE
[script.skinshortcuts] 1.0.17

### PR DESCRIPTION
**Pull request by marcelveldt**
### Description
- Some minor changes - targetted for Jarvis+ only
- Basic support for Kodi Leia (while we're still working on a refactored version for Leia+ only)
### Checklist:
- [X ] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X ] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [X ] Each add-on submission should be a single commit with using the following style: [sciprt.foo.bar] v1.0.0

### CODE REVIEW

**Comment by marcelveldt** [comment]
any word on this ?

**Comment by Tadly** [comment]
With the rest I'm fine with :)

File: script.skinshortcuts/addon.xml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.skinshortcuts" name="Skin Shortcuts" version="1.0.17" provider-name="BigNoid,marcelveldt">
+	<requires>
+		<import addon="xbmc.python" version="2.20.0"/>
```
Comment (Tadly) [inline script.skinshortcuts/addon.xml]: Sorry, forgot to submit the review (again.. I'll learn to remember at some point)

**Comment by Razzeee** [comment]
Finally found some time to look into this. I think we can merge this.

One additional thing, for @marcelveldt looking at the diff makes me think you might want to look into moving certain functions like logging or this:
````
ISEMPTY = "IsEmpty"
if int( KODIVERSION ) >= 17:
    ISEMPTY = "String.IsEmpty"

```
to a seperate file and just reference it via an import. Keeps you from having duplicated code.

**Comment by marcelveldt** [comment]
Thanks for the review and the merge!
The current code for Kodi 17 is just temporary, a refactor of the addon is already in the making.

**Pull request merged**